apps/researcher/src/lib/encode-uri-identifier.ts

```diff
@@ -0,0 +1,10 @@
+// Linked data uses URIs as identifiers.
+// To use the URIs in dynamic routes, we need to encode them first.
+
+export function encodeUriIdentifier(uri: string) {
+  return encodeURIComponent(uri.replace(/\./g, '%2E'));
+}
+
+export function decodeUriIdentifier(routeSegment: string) {
```

Comment:
- `encodeUriIdentifier` uses `uri` as param, this function `routeSegment`. Use a uniform name (e.g. `uri`)?
- This function uses a `replaceAll()` but the `encodeUriIdentifier` a `replace` with a `g` flag. Is it possible to harmonize these?